### DISCUSSION TITLE
api.vanjs.org

### DISCUSSION THREAD

**Discussion by sirenkovladd** [vanjs-org/van#335] [General]
Is it possible to make the code of this page public and add documentation to it?

**Comment by Tao-VanJS** [comment]
Codebase: https://github.com/Tao-VanJS/vanjs-preview.

You can refer to this [file](https://github.com/Tao-VanJS/vanjs-preview/blob/main/schema.yaml) for documentation.

**Reply by sirenkovladd** [reply]
Thanks, I might leave this discussion if anyone else has questions about this page.
